### PR TITLE
removed endpoint toggle

### PR DESCRIPTION
**Pull request by msho-student**

### CODE REVIEW

**Comment by msho-student** [comment]
lgtm

**Review by asharma381 — APPROVED** [review]
LGTM!

File: src/App.jsx [hunk 1]
```diff
@@ -36,8 +34,8 @@ function App() {
   // const endpoints = ['get_s1_ordered', 'get_hit/null/render_worker_stats'];
 
   const getHit = () => {
-    const endpoint = useAltGetHit.current.value ? 'get_s1_ordered' : 'get_hit/null/render_worker_stats';
-    // fetch('https://the.mturk.monster:50000/api/get_hit/null/render_worker_stats')
+    const endpoint = 'get_s1_ordered';
+    // const endpoint = 'get_hit/null/render_worker_stats';
```
Comment (asharma381) [inline src/App.jsx:38]: remove comment

File: src/App.jsx [hunk 2]
```diff
@@ -36,8 +34,8 @@ function App() {
   // const endpoints = ['get_s1_ordered', 'get_hit/null/render_worker_stats'];
```
Comment (asharma381) [inline src/App.jsx:34]: comment

**Pull request merged**